### PR TITLE
Updating modules dependencies

### PR DESCRIPTION
**Pull request by jamengual**
## what
* Updating modules dependencies
* Updating variable output names changes from ECR upstream module

## why
* Many bug fixes and improvements

### CODE REVIEW

File: test/Makefile [hunk 1]
```diff
@@ -4,6 +4,9 @@ TEST_HARNESS_PATH = $(realpath .test-harness)
 BATS_ARGS ?= --tap
 BATS_LOG ?= test.log
 
+# Expose GitHub Token as github_webhooks_token variable so that SSM does not fail
+export TF_VAR_github_webhooks_token ?= $(GITHUB_TOKEN)
```
Comment (osterman) [inline test/Makefile:8]: Oh, this is the wrong `Makefile`. Move it to `test/src/Makefile`

**Comment by osterman** [comment]
Update the PR description to explain why the renaming of the variables.

File: test/Makefile (hunk 1 above)
Comment (osterman) [reply test/Makefile:8]: (this is the `Makefile` for the `bats` automated testing. The tests are failing with `terratest`)

The `bats` is currently failing due to listing. run `terraform fmt` for that.

Comment (jamengual) [reply test/Makefile:8]: I see ok

**Pull request merged**